server/index.js

```diff
@@ -27,21 +27,71 @@ app.get('/api/sneakers', async (req, res, next) => {
   }
 });
 
-// app.post('/api/sneakers', async (req, res, next) => {
-//   const searchTerm = req.body;
-//   const sql = `
-//   select *
-//   from "sneakers"
-//   where "brand" = $1;
-//   `;
-//   const param = [searchTerm];
-//   try {
-//     const sneakers = await db.query(sql, param);
-//     res.json(sneakers);
-//   } catch (err) {
-//     next(err);
-//   }
-// });
+let ClientError;
```

Comment:
You should have a ClientError class defined somewhere. Check your node exercises, like the one about error middleware.